test/EFCore.PG.FunctionalTests/Query/RangeQueryNpgsqlTest.cs

```diff
@@ -498,7 +498,7 @@ public void UserDefinedSchemaQualified()
             using (var context = Fixture.CreateContext())
             {
                 var e = context.RangeTestEntities.Single(x => x.SchemaRange == NpgsqlRange<double>.Parse("(0,10)"));
-                AssertContainsSql("WHERE x.\"SchemaRange\" = @__Parse_0");
+                AssertContainsSql("WHERE x.\"SchemaRange\" = '(0,10)'::test.\"Schema_Range\"");
```

Comment:
That looks great!

Reply:
Yeah I was impressed too :) It's a bit mind-boggling to have an ORM that's generic and customizable enough to produce this kind of literal...